pymultimatic/api/payloads_senso.py

```diff
@@ -24,7 +24,7 @@ def zone_temperature_setpoint(temperature: float) -> Dict[str, Any]:
     """Payload used to set target temperature for
     :class:`~pymultimatic.model.component.Zone`.
     """
-    return {"temperature_setpoint": temperature}
+    return {"manual_mode_temperature_setpoint": temperature}
```

Comment:
Is it really the same as setting the "target high" temperature ?

I think we already spoke about the TIMED_CONTROLLED way of working but I forget :(.

How do you actually define the target high temperature for time TIMED_CONTROLLED ?

Reply:
For the TIMED_CONTROLLED mode, we define the temperature "setback_temperature_setpoint" for the unscheduled periods and indicate the "target high" with the API "_ZONE_HEATING_TIMEPROGRAM" for each day with the start and end time and the temperature.
Here is my test:
```python
await manager.set_zone_heating_operating_mode(zone_id, OperatingModes.TIME_CONTROLLED)
        await manager.set_zone_heating_setback_temperature(zone_id, 19.0)
        await manager._call_api(manager.urls.zone_heating_timeprogram, params={"id": zone_id},
                                payload={
                    "monday": [
                        {
                            "start_time": "04:30",
                            "end_time": "22:00",
                            "setpoint": 21.0
                        }
                    ],
                    "tuesday": [
                        {
                            "start_time": "04:30",
                            "end_time": "22:00",
                            "setpoint": 21.0
                        }
                    ],
                    "wednesday": [
                        {
                            "start_time": "04:30",
                            "end_time": "22:00",
                            "setpoint": 21.0
                        }
                    ],
                    "thursday": [
                        {
                            "start_time": "04:30",
                            "end_time": "22:00",
                            "setpoint": 21.0
                        }
                    ],
                    "friday": [
                        {
                            "start_time": "04:30",
                            "end_time": "22:00",
                            "setpoint": 21.0
                        }
                    ],
                    "saturday": [
                        {
                            "start_time": "07:00",
                            "end_time": "23:00",
                            "setpoint": 21.0
                        }
                    ],
                    "sunday": [
                        {
                            "start_time": "07:00",
                            "end_time": "22:00",
                            "setpoint": 21.0
                        }
                    ]
                })
```
And we get:
![IMG_6AD333A1D6F5-1](https://user-images.githubusercontent.com/5923066/226459525-69d6b8e5-2a06-40d7-a012-46246ec67a97.jpeg)

I add to my todo list the implementation of TIMEPROGRAM in the manager ;)